test_munch.py

```diff
@@ -111,7 +111,11 @@ def test_fromDict():
 def test_copy():
```

Comment:
Please add a test for copying DefaultMunch's and making sure it preserves the type and default value